ckan/tests/factories.py

```diff
@@ -310,8 +310,8 @@ class Meta:
     type = "organization"
 
 
-class Dataset(CKANFactory):
-    """A factory class for creating CKAN datasets."""
+class Package(CKANFactory):
```

Comment:
do we need two names for the same thing in tests?

Reply:
So you think we should switch to the 'package' naming internally in ckan. I'm not against that but it's something we should bring up in a dev meeting and get a consensus.